holmes/core/conversations.py

```diff
@@ -340,5 +353,4 @@ def build_chat_messages(
         ai, conversation_history_without_tools, number_of_tools
     )
     truncate_tool_messages(conversation_history, tool_size)
-
     return conversation_history
```

Comment:
The conversation history is including now the global instructions this means that this what the user will see in the history chat?